autobahn/wamp/message.py

```diff
@@ -92,14 +95,69 @@
 # See: http://cr.yp.to/highspeed/coolnacl-20120725.pdf
 PAYLOAD_ENC_CRYPTO_BOX = u'cryptobox'
 
+# Payload transparency identifier for MQTT payloads (which are arbitrary binary).
+PAYLOAD_ENC_MQTT = u'mqtt'
+
+# Payload transparency algorithm identifiers from the WAMP spec.
+PAYLOAD_ENC_STANDARD_IDENTIFIERS = [PAYLOAD_ENC_CRYPTO_BOX, PAYLOAD_ENC_MQTT]
+
+# Payload transparency serializer identifiers from the WAMP spec.
+PAYLOAD_ENC_STANDARD_SERIALIZERS = [u'json', u'msgpack', u'cbor', u'ubjson']
+
+
+def is_valid_enc_algo(enc_algo):
+    """
+    For WAMP payload transparency mode, check if the provided ``enc_algo``
+    identifier in the WAMP message is a valid one.
+
+    Currently, the only standard defined identifier are
+
+    * ``u"cryptobox"``
+    * ``u"mqtt"``
+
+    Users can select arbitrary identifiers too, but these MUST start with ``u"x_"``.
+
+    :param enc_algo: The payload transparency algorithm identifier to check.
+    :type enc_algo: str
+
+    :returns: Returns ``True`` if and only if the payload transparency
+        algorithm identifier is valid.
+    :rtype: bool
+    """
+    return type(enc_algo) == six.text_type and (enc_algo in PAYLOAD_ENC_STANDARD_IDENTIFIERS or _CUSTOM_ATTRIBUTE.match(enc_algo))
+
```

Comment:
I think it's slightly "more correct" to do `isinstance(enc_algo, six.text_type)` -- but that might be a better as a "change the whole codebase" PR so we're at least consistent ;)